Software/lib/GNSE_HAL/GNSE_bm.c

```diff
@@ -22,6 +22,42 @@
 
 #include "GNSE_bm.h"
 
+/**
+  * @brief Deinitialises the hardware for reading the battery voltage level
+  * @param none
+  * @return BM_op_result_t
+  */
+BM_op_result_t GNSE_BM_DeInit(void)
```

Comment:
The order of functions is Init and then de-init, please adjust all new functions.